src/control/cmd/dmg/main.go

```diff
@@ -276,18 +297,7 @@ func parseOpts(args []string, opts *cliOptions, ctlClient control.Invoker, conns
 			log.Info(connStates.String())
 			wantsConn.setConns(conns)
 		}
-
-		ctlClient.SetClientConfig(ctlClientCfg)
-		if clientCmd, ok := cmd.(ctlClientUser); ok {
-			clientCmd.setClient(ctlClient)
-			if opts.HostList != "" {
-				clientCmd.setHostList(strings.Split(opts.HostList, ","))
-			}
-		}
-
-		if cfgCmd, ok := cmd.(cmdConfigSetter); ok {
-			cfgCmd.setConfig(config)
-		}
+		// END DEPRECATED CLIENT SETUP (DAOS-4632)
```

Comment:
Since you're refactoring this now, could you break the deprecated setup out into a helper function? Makes it even easier to know what to remove later.

Reply:
Could do that, but I'm planning on nuking it all relatively soon anyhow.